Fixed Typos (#259)
robocomp/robocomp: CONTRIBUTING.md

### Reporting an Issue

> - Make sure you test against the latest version (of the highly unstable branch). It is possible we 

Reviewer: ```suggestion
- Make sure you test against the latest version (of the development branch). It is possible we 
```

be sure to update the `Readme.md` installation tutorial to include your changes.

- If you plan to make big changes, ensure you are not breaking Robocomp's core functionalities.

> Even though Travis tests the build, you could break an internal function. 

Reviewer: ```suggestion
Even though CI tests the build, you could break an internal function. 
```

> Please ensure you test new features locally at first.

Reviewer: ```suggestion
Please ensure you test new features locally at first and add some unit tests for the new functionalities.
```

Reviewer: Thank you for your contribution and for pointing to this piece of the documentation.

Author: Welcome, @orensbruli.
